cypher-shell/src/test/java/org/neo4j/shell/prettyprint/PrettyPrinterTest.java

```diff
@@ -160,6 +158,36 @@ public void prettyPrintList() throws Exception {
         assertThat(actual, is("col1, col2\n[val1_1, val1_2], [val2_1]\n[val2_1]"));
     }
 
+    @Test
+    public void prettyPrintMaps() throws Exception {
+        checkMapForPrettyPrint(map(), "map\n{}");
```

Comment:
Use `String.format("map%n{}"` instead to avoid breaking on windows.

Reply:
Fair point. Changes have been amended.